apps/testingaccessibility/src/pages/api/auth/[...nextauth].ts

```diff
@@ -30,9 +30,12 @@ export const sendVerificationRequest = async (params: {
     url,
     provider: {server, from},
   } = params
-  const {host} = new URL(url)
+  const rewriteUrl = new URL(url)
   const transport = createTransport(server)
   const {getUserByEmail} = getSdk()
+  const {host} = rewriteUrl
+
+  rewriteUrl.pathname = '/api/callback'
```

Comment:
rewrites the url going to the inbox with our userland route that does a client-side redirect